--- /dev/null
+++ ../src/Common/Internal/ServiceRestProxy.php
@@ -77,7 +77,7 @@
         array $options = []
     ) {
         $primaryUri   = Utilities::appendDelimiter($primaryUri, '/');
-        $secondaryUri = Utilities::appendDelimiter($secondaryUri, '/');
+        $secondaryUri = Utilities::appendDelimiter($secondaryUri ?? '', '/');
 
         $dataSerializer = new XmlSerializer();
         parent::__construct($dataSerializer);
